docs/revsets.md

```diff
@@ -34,9 +34,9 @@ typically only one visible commit with a given change ID). A unique prefix of
 the full change ID can also be used. It is an error to use a non-unique prefix.
```

Comment:
Should there be a mention here somewhere about adding spaces around (or parentheses around the identifier preceding) `+`/`-` to force parsing as an operator? (Assuming that's actually the case, of course...)

Reply:
Those two operators are only postfix operators right now and no space is needed.

Reply:
Ah, gotcha, thanks for clarifying that 😄